pkg/inmemorychannel/message_dispatcher_benchmark_test.go

```diff
@@ -77,7 +77,7 @@ func BenchmarkDispatcher_dispatch_ok_through_2_channels(b *testing.B) {
 	}
 
 	// Let's mock this stuff!
-	httpSender, err := kncloudevents.NewHTTPMessageSenderWithTarget(channelA.URL.String())
+	request, err := kncloudevents.NewCloudEventRequest(context, channelA.URL.String())
 	if err != nil {
 		b.Fatal(err)
 	}
```

Comment:
```suggestion
```
Ah. I didn't see, that we switched to it in line 103 already :facepalm: 
Sorry for the miscommunication. :(